resources/js/views/Locations/SortLocations.vue

```diff
@@ -194,7 +201,7 @@ export default {
 
 	.locations-main {
 		background-color: #23d160;
-		min-height: 100%;background-color: #23d160; 
+		min-height: 100%;background-color: #23d160;
```

Comment:
background-color is duplicated here

Reply:
It's now fixed from merging staging here.